### PR TITLE
mvnd --install feature

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@gnodet could you please review?

I'd try to create the first release after this.

File: client/src/main/java/org/jboss/fuse/mvnd/client/DefaultClient.java [hunk 1]
```diff
@@ -73,14 +81,36 @@ public static void main(String[] argv) throws Exception {
         }
     }
 
+    private static void install(boolean overwrite) {
+        final Properties buildProps = loadBuildProperties();
+        final String version = buildProps.getProperty("version");
+        final String rawZipUri = System.getProperty("mvnd.zip.uri", "https://github.com/ppalaga/mvnd/releases/download/"+ version +"/mvnd-" + version + ".zip");
```
Comment (gnodet) [inline client/src/main/java/org/jboss/fuse/mvnd/client/DefaultClient.java]: Shouldn't this be modified to point to the `mvndaemon/mvnd` repo instead ?

File: client/src/main/java/org/jboss/fuse/mvnd/client/Installer.java [hunk 2]
```diff
@@ -0,0 +1,194 @@
+package org.jboss.fuse.mvnd.client;
```
Comment (gnodet) [inline client/src/main/java/org/jboss/fuse/mvnd/client/Installer.java:1]: Missing header ...

File: client/src/main/java/org/jboss/fuse/mvnd/client/DefaultClient.java (hunk 1 above)
Comment (ppalaga) [reply client/src/main/java/org/jboss/fuse/mvnd/client/DefaultClient.java]: I changed it in one of the subsequent commits https://github.com/mvndaemon/mvnd/pull/30/files#diff-86f08db3461cd372b91b0c70cd8606c5R85

File: client/src/main/java/org/jboss/fuse/mvnd/client/Installer.java (hunk 2 above)
Comment (ppalaga) [reply client/src/main/java/org/jboss/fuse/mvnd/client/Installer.java:1]: +1

**Comment by ppalaga** [comment]
4461d67 added the missing license headers.

**Comment by ppalaga** [comment]
The CI passed. I guess this is good to merge.

**Pull request merged**